runbotics/runbotics-orchestrator-ui/src/src-app/credentials/useRequiredCredentialTypes.ts

```diff
@@ -0,0 +1,15 @@
+import { useMemo } from 'react';
+
+import { getRequiredCredentialsTypesInProcess } from '#src-app/credentials/getRequiredCredentialsTypesInProcess';
+import { useSelector } from '#src-app/store';
+import { processSelector } from '#src-app/store/slices/Process';
+
+export const useRequiredCredentialTypes = () => {
```

Comment:
return type (unless it's its deducted by TS) should be added

Reply:
added